README.md

```diff
@@ -217,112 +222,113 @@ created during these steps with your initial SNS developer neurons).
    [Test canister](https://github.com/dfinity/sns-testing#test-canister)
    for further details) which can be thought of as a placeholder
    for your dapp.
-2. Run the script
+
+2. Give control of your dapp canisters to NNS:
    ```bash
-   ./deploy_sns.sh <config-path>  # from Bash
+   ./let_nns_control_dapp.sh  # from Bash
```

Comment:
Shouldn't a script like this take an argument that is the canister_id of your dapp? Seems like we've departed from how the flow actually works if you're using `sns prepare-canisters add-nns-root <CANISTER_ID>`

Reply:
The canister IDs are fetched from the yaml file which is an optional parameter of the script.

Reply:
Hi @DanielThurau 

> Seems like we've departed from how the flow actually works if you're using sns prepare-canisters add-nns-root <CANISTER_ID>

I don't quite understand your concern here. AFAIK, the `let_nns_control_dapp.sh` script was created by @daniel-wong-dfinity-org specifically for 1-prop, I've not changed it in this PR.

Reply:
> Shouldn't a script like this take an argument that is the canister_id of your dapp?

As Martin points out, this is now the case, but the argument is optional. Would you prefer to make it non-optional?

Reply:
Let me also point out explicitly that the optional argument is a path to a yaml file, not the canister_id of your dapp.

Reply:
Right. Note that the new yaml file specifies the canister IDs of the dapp's canisters, so a script doesn't need to ask for those again if it already known the path to the yaml file.